### PR TITLE
Handler Rewrite

### LINKED ISSUE #2: Test files
We need to check for a few things:
* Check for invalid token
* Check for snowflake client id for application commands endpoint
* Check for invalid commands uploaded
* Check for if valid commands are uploaded that Discord API will receive them proper

Jest: https://jestjs.io/docs/getting-started

### PR DESCRIPTION
**Pull request by bowenjw**
# New Handlers
Rewrote command, interaction and event handlers for easy of use.

## Updated `packages.json`
updated all modules to updated versions
Changes brake existing code and version updated to `0.4.0`

### CODE REVIEW

**Comment by bowenjw** [comment]
@Lewdcario please look at this when you get a chance

File: README.md [hunk 1]
```diff
@@ -82,11 +81,11 @@ export default new ChatInputCommand()
 
 ### How to create interaction object
 
-Interaction objects are a generic and can be used for any object the following is the example of a button interaction. Button interactions should be placed in the Directory path set in the `buttonPath`
+Interaction objects are a genaric and can be used for any object the following is the example of a button interaction. Button interactions should be placed in the Dirctory path set in the `buttonPath`
```
Comment (Lewdcario) [inline README.md]: Dirctory -> directory

File: jest.config.ts [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+	// This option allows use of a custom test runner
+	// testRunner: "jest-circus/runner",
+
+	// A map from regular expressions to paths to transformers
+	transform: {
+		'^.+\\.tsx?$': ['ts-jest', {
+			isolatedModules: true,
+			tsconfig: 'tsconfig.json'
+		}]
+	},
```
Comment (Lewdcario) [inline jest.config.ts]: This was my bad but can you remove the "x" here from tsx, this was copied over from my nextjs project

**Review by Lewdcario — CHANGES_REQUESTED** [review]
Looks very good, just nitpicking a little bit on spelling

**Review by Lewdcario — APPROVED** [review]

**Pull request merged**